README.md

```diff
@@ -415,6 +446,16 @@ This method makes a call to the
 [/{db}/_design/{ddoc}/_view/{view}](http://docs.couchdb.org/en/1.6.1/api/ddoc/views.html#get--db-_design-ddoc-_view-view)
 endpoint, and the builder supports all the relevant options.
 
+Similarly, to query the total age of Persons in the document using the
+`totalAgeView` builder we can do:
+
+```Scala
+totalAgeView.queryWithReduce[Int]
```

Comment:
I see, you are right. It would be nice to add the type parameters to CouchView, so that the types are declared closer to the definition of the map and reduce functions. Then, it wouldn't be necessary to specify them later. I need to think a bit more about it and see how it might work.